docs/source/sharing.rst

```diff
@@ -0,0 +1,93 @@
+Sharing
```

Comment:
I agree it would be premature; it makes more sense to have plone.api terminology in line with plone proper. Should the change happen later, there's probably ways to have one endpoint be an alias to any new terminology that may come in.